example_app_generator/generate_app.rb

```diff
@@ -1,7 +1,7 @@
 rspec_rails_repo_path = File.expand_path("../../", __FILE__)
 rspec_dependencies_gemfile = File.join(rspec_rails_repo_path, 'Gemfile-rspec-dependencies')
 rails_dependencies_gemfile = File.join(rspec_rails_repo_path, 'Gemfile-rails-dependencies')
-bundle_install_path = File.join(rspec_rails_repo_path, '..', 'bundle')
+bundle_install_path = File.join(rspec_rails_repo_path, 'vendor', 'bundle')
```

Comment:
Many of the core team setup their environment with a shared bundle path across all repos. This path is the directory containing the `rspec-rails` repo. Additionally, the Travis CI setup does this as well.

I do understand that this is likely not ideal for any system outside of the above subset. Perhaps we can come up with a better solution to keep all environments happy.